base_jsonify/models/ir_exports_resolver.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+from odoo.tools.safe_eval import safe_eval
+
+help_message = [
+    "Compute the result from 'value' by setting the variable 'result'.",
+    "For fields resolvers:",
+    ":param name: name of the field",
+    ":param value: value of the field",
+    ":param field_type: type of the field",
+    "For global resolvers:",
+    ":param value: json dict",
+    ":param record: the record",
+]
+
+
+class FieldResolver(models.Model):
+    """Arbitrary function to process a field or a dict at export time.
+    """
+
+    _name = "ir.exports.resolver"
+    _description = "Resolver"
+
+    name = fields.Char()
+    type = fields.Selection([("field", "Field"), ("global", "Global")])
+    python_code = fields.Text(
+        string="Python Code",
+        default="\n# ".join(["result = value"] + help_message),
```

Comment:
`result = value` shouldn't stay at the end?

Reply:
I agress with @simahawk